PC-VRPTW/RC1_10_8.vrp

```diff
@@ -0,0 +1,4019 @@
+NAME : RC1_10_8
+TYPE : CVRPTW
+DIMENSION : 1001
+VEHICLES : 90
+CAPACITY : 200
+SERVICE_TIME : 10
```

Comment:
How about making a `SERVICE_TIME_SECTION` of this with homogeneous service times, with 0 for the depot?

(I think that service times were never part of the original VRPLIB format because it was used for CVRP. But it makes more sense to me to have all client data in data sections instead of a keyword-value pair.)

Reply:
I considered that, but this is how this stuff came from Helsgaun's website and I didn't want to go around modifying it much beyond adding a prize section.

Reply:
OK, this works for me since we changed `read.py` anyway in PyVRP.

Reply:
I figured we needed that change because these LKH3 instances are pretty well-known, and we ideally should be able to parse them if at all possible.